docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -7,15 +7,19 @@ Vitamin A Deficiency
 Disease Description
 -------------------
 
-Vitamin A deficiency (VAD) is a lack of vitamin A in blood and tissues.
-Vitamin A deficiency is considered as one of the most serious public health concerns in developing countries
-and can contribute directly or indirectly to disability.[1]
+Vitamin A deficiency (VAD) is a lack of vitamin A in blood and tissues. Vitamin
+A deficiency is considered as one of the most serious public health concerns in
+developing countries and can contribute directly or indirectly to disability.[1]
 
 GBD 2017 Modeling Strategy
 ------------------------------------
-In Global Burden of Disease (GBD) 2017, VAD exposure definition is proportion of the population with serum retinol concentration <0·7 μmol/L.
-Like iron deficiency, the cause VAD is also a population attributable fraction (PAF) of 1 cause with the VAD risk factor. That is, 100% of the VAD cases are attributable
-to the VAD risk factor. VAD Risk exposure and VAD cause prevalence data are the same.[2]
+
+In Global Burden of Disease (GBD) 2017, VAD exposure definition is proportion of
+the population with serum retinol concentration <0·7 μmol/L. Like iron
+deficiency, the cause VAD is also a population attributable fraction (PAF) of 1
+cause with the VAD risk factor. That is, 100% of the VAD cases are attributable
+to the VAD risk factor. VAD Risk exposure and VAD cause prevalence data are the
```

Comment:
Does this mean that all data that goes into the GBD exposure model is the same as the data that goes into the GBD prevalence model, or that everyone with VAD exposure also has the VAD cause? (i.e., is this about GBD model inputs or outputs?)

Reply:
This means the GBD outputs are the same. There is only one VAD model, not separate models for the risk and cause. I can add some wording to clarify this.